src/style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+
+  --secondary-color: #616070;
+}
+
+.card {
```

Comment:
Follow the checklist 
<img width="597" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/99403198/2d9024b2-5578-4fd8-a2b4-2b05f90e0507">